private/multiset.rkt

```diff
@@ -103,6 +104,9 @@
 
 (define (list->multiset lst) (apply multiset lst))
 
+(define (sequence->multiset seq)
+  (for/multiset ([v seq]) v))
```

Comment:
Optional: this can be `(reduce-all into-multiset seq)` instead, which I find a little simpler. Nothing wrong with the `for/multiset` loop though.

Reply:
👍 I was trying to divine what your preferred style was for this.  I pretty much just copied the `multiset` function.